libs/application/templates/id-card/src/fields/SecondGuardianEmailAndPhone/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { FC, useState } from 'react'
+import { FieldBaseProps } from '@island.is/application/types'
+import { GridRow, GridColumn } from '@island.is/island-ui/core'
+import {
+  InputController,
+  PhoneInputController,
+} from '@island.is/shared/form-fields'
+import { applicantInformation, error } from '../../lib/messages'
+import { useLocale } from '@island.is/localization'
+import { useFormContext } from 'react-hook-form'
+import { getValueViaPath } from '@island.is/application/core'
+import { Routes } from '../../lib/constants'
+
+type FieldError =
+  | {
+      secondGuardianInformation?: {
+        email?: string
+        phoneNumber?: string
+      }
+      applicantInformation?: string
+    }
+  | undefined
+
+export const SecondGuardianEmailAndPhone: FC<
```

Comment:
Why does this have to be a custom component?

Reply:
Because if the user goes back, chooses another child with other second guardian, we will need have to clear out the email and phone for the new guardian, I could not do it on the page before, because it is required. If I would be able to remove this item from answers I would not have to do this.